system/ee/EllisLab/ExpressionEngine/Service/Model/Column/ColumnObject/Timestamp.php

```diff
@@ -8,7 +8,7 @@
  * @license   https://expressionengine.com/license Licensed under Apache License, Version 2.0
  */
 
-namespace EllisLab\ExpressionEngine\Service\Model\Column\Object;
+namespace EllisLab\ExpressionEngine\Service\Model\Column\ColumnObject;
```

Comment:
What is the reason / benefit for changing this from `Object` to `ColumnObject`?

Reply:
`Object` is a loose protected term in PHP7. Throws a warning now, which means it will be a fatal error in PHP8.